src/app/window-actions.ts

```diff
@@ -194,3 +207,65 @@ export const removeWindowEventListener = (window: BrowserWindow): void => {
     window.removeListener('leave-full-screen', windowUnmaximized);
     window.removeListener('unmaximize', windowUnmaximized);
 };
+
+/**
+ * Verifies the permission and display a
+ * dialog if the action is not enabled
+ *
+ * @param permission {boolean} - config value to a specific permission
+ * @param message {string} - custom message displayed to the user
+ * @param callback {function}
+ */
+export const handleSessionPermissions = (permission: boolean, message: string, callback: (permission: boolean) => void): void => {
+    logger.info(`window-action: permission is ->`, { type: message, permission });
+
+    if (!permission) {
+        const fullMessage = `${i18n.t('Your administrator has disabled')()} ${message}. ${i18n.t('Please contact your admin for help')()}`;
```

Comment:
we'll need a space after `disabled`

Reply:
@VishwasShashidhar There is already space after `disabled`

Reply:
Ah! Templating, yes, thanks :)